code/game/machinery/doors/airlock.dm

```diff
@@ -488,18 +478,23 @@ There are 9 wires.
 	return src.isWireCut(AIRLOCK_WIRE_BACKUP_POWER1) || src.isWireCut(AIRLOCK_WIRE_BACKUP_POWER2)
 
 /obj/machinery/door/airlock/proc/loseMainPower()
-	main_power_lost_until = mainPowerCablesCut() ? -1 : world.time + SecondsToTicks(60)
+	main_power_lost_until = mainPowerCablesCut() ? -1 : SecondsToTicks(60)
+	if(main_power_lost_until > 0)
+		spawn(main_power_lost_until) regainMainPower()
```

Comment:
https://github.com/discordia-space/CEV-Eris/pull/6527#discussion_r710784776
And we don't use linear blocks like if() action(), for() action(), spawn() action()